### PR TITLE
Iceberg connector document update about metadata tables

### PR DESCRIPTION
**Pull request by agrawalreetika**
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-docs/src/main/sphinx/connector/iceberg.rst [hunk 1]
```diff
@@ -282,6 +282,41 @@ original state.
...
+
+* ``$properties`` : General properties of given table
+.. code-block:: sql
+
+    SELECT * FROM "ctas_nation$properties";
+
+* ``$history`` : History about change to the table state
+.. code-block:: sql
+
+    SELECT * FROM "ctas_nation$properties";
```
Comment (majetideepak) [inline presto-docs/src/main/sphinx/connector/iceberg.rst:299]: `ctas_nation$history`

File: presto-docs/src/main/sphinx/connector/iceberg.rst [hunk 2]
```diff
@@ -282,6 +282,41 @@ original state.
      comment   | varchar |       |
     (4 rows)
 
+Extra Hidden Metadata Tables
+----------------------------
+
+The Iceberg connector exposes extra hidden metadata tables. You can query these
+as a part of SQL query by appending name with the table.
+
+* ``$properties`` : General properties of given table
```
Comment (majetideepak) [inline presto-docs/src/main/sphinx/connector/iceberg.rst:291]: General properties of the given table

File: presto-docs/src/main/sphinx/connector/iceberg.rst [hunk 3]
```diff
@@ -282,6 +282,41 @@ original state.
...
+.. code-block:: sql
+
+    SELECT * FROM "ctas_nation$properties";
+
+* ``$history`` : History about change to the table state
```
Comment (majetideepak) [inline presto-docs/src/main/sphinx/connector/iceberg.rst:296]: History of table state changes

File: presto-docs/src/main/sphinx/connector/iceberg.rst [hunk 4]
```diff
@@ -282,6 +282,41 @@ original state.
...
+.. code-block:: sql
+
+    SELECT * FROM "ctas_nation$properties";
+
+* ``$snapshots`` : Details about the table snapshots
```
Comment (majetideepak) [inline presto-docs/src/main/sphinx/connector/iceberg.rst:301]: Is it possible to add more information about the details here and below?

**Comment by agrawalreetika** [comment]
HI @majetideepak, Thanks for your review. I have made changes as per the review comments, please review.

**Review by majetideepak — APPROVED** [review]
Thanks, @agrawalreetika

**Review by pratyakshsharma — APPROVED** [review]
LGTM as well

**Comment by agrawalreetika** [comment]
Hi @tdcmeehan, Could you please review this?
If it looks to you as well, then we can merge this documentation.

File: presto-docs/src/main/sphinx/connector/iceberg.rst [hunk 5]
```diff
@@ -282,6 +282,79 @@ original state.
...
+    SELECT * FROM "ctas_nation$snapshots";
+
+.. code-block:: text
+
+                 committed_at             |     snapshot_id     | parent_id | operation |                                                  manifest_list                                           |                                                                                 summary
+    --------------------------------------+---------------------+-----------+-----------+----------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+    2022-11-25 20:56:31.784 Asia/Kolkata | 7606232158543069775 | NULL      | append     | s3://my-bucket/ctas_nation/metadata/snap-7606232158543069775-1-395a2cad-b244-409b-b030-cc44949e5a4e.avro | {changed-partition-count=1, added-data-files=1, total-equality-deletes=0, added-records=25, total-position-deletes=0, added-files-size=1648, total-delete-files=0, total-files-size=1648, total-records=25, total-data-files=1}
```
Comment (tdcmeehan) [inline presto-docs/src/main/sphinx/connector/iceberg.rst:322]: Screenshot 2023-06-12 at 10 01 26 AM

Looks like formatting is slightly off

**Review by tdcmeehan — COMMENTED** [review]
What do you think about linking these things to the Iceberg spec?  For example, in the snapshots section, adding a link here: https://iceberg.apache.org/spec/#snapshots

File: presto-docs/src/main/sphinx/connector/iceberg.rst (hunk 5 above)
Comment (agrawalreetika) [reply presto-docs/src/main/sphinx/connector/iceberg.rst]: Corrected. And added spec documentation for snapshots & manifests

**Review by tdcmeehan — APPROVED** [review]

**Pull request merged**